src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -132,6 +172,14 @@ public boolean requiresReference() {
     public void onTraversalStart() {
         final SAMSequenceDictionary sequenceDictionary = getBestAvailableSequenceDictionary();
 
+        if (binLength > 0 && minimumBinLength > binLength) {
+            throw new UserException.BadInput(
+                    String.format("%s (%s) must be less or equal than %s (%s)", MINIMUM_BIN_LENGTH_LONG_NAME,
+                            minimumBinLength, BIN_LENGTH_LONG_NAME, binLength));
+        } else if (grid && binLength <= 0) {
+            throw new UserException.BadInput("when requested gridded bin you must speficy a bin-length larger than 0");
```

Comment:
When requesting gridded bins, you must specify a bin-length greater than 0.

Reply:
Perhaps also extract these checks to a validateArguments method as is done in other CNV tools.